multipaz-compose/src/commonMain/kotlin/org/multipaz/compose/camera/CameraFrame.kt

```diff
@@ -0,0 +1,16 @@
+package org.multipaz.compose.camera
+
+import androidx.compose.ui.graphics.ImageBitmap
+
+/**
+ * Common Camera Frame data object. No transformations applied to the original image data.
+ *
+ * @param data The image data converted from the native format to common KMP image bitmap format.
+ * @param width The image width (pixels).
+ * @param height The image height (pixels).
+ */
+data class CameraFrame(
+    val data: ImageBitmap,
+    val width: Int,
+    val height: Int
+)
```

Comment:
No real point in this class if it doesn't have a platform-specific type like e.g. `ImageProxy` on Android. So just remove this and use `ImageBitmap` instead.